diff --git a/viewer/viewer.js b/viewer/viewer.js
index b691b81..bb9e3dd 100644
--- a/viewer/viewer.js
+++ b/viewer/viewer.js
@@ -7377,11 +7377,6 @@
           return;
         }
 
-        // get the size of the query if it is being restarted
-        if (hunt.lastPacketTime) {
-          query.size = hunt.totalSessions - hunt.searchedSessions;
-        }
-
         lookupQueryItems(query.query.bool.filter, (lerr) => {
           query.query.bool.filter[0] = {
             range: {
