diff --git a/viewer/viewer.js b/viewer/viewer.js
index fb480e56..f6f577fe 100644
--- a/viewer/viewer.js
+++ b/viewer/viewer.js
@@ -7255,11 +7255,6 @@ function processHuntJob (huntId, hunt) {
           return;
         }
 
-        // get the size of the query if it is being restarted
-        if (hunt.lastPacketTime) {
-          query.size = hunt.totalSessions - hunt.searchedSessions;
-        }
-
         lookupQueryItems(query.query.bool.filter, (lerr) => {
           query.query.bool.filter[0] = {
             range: {
