diff --git a/viewer/viewer.js b/viewer/viewer.js
index bb9e3dd..6f88294 100644
--- a/viewer/viewer.js
+++ b/viewer/viewer.js
@@ -899,6 +899,8 @@
     var info = url.parse(viewUrl);
     info.path = req.url;
     info.agent = (client === http?internals.httpAgent:internals.httpsAgent);
+    // use internals.esQueryTimeout, but convert it to milliseconds
+    info.timeout = parseInt(Config.get("elasticsearchTimeout", 300), 10) * 1000;
     addAuth(info, req.user, req.params.nodeName);
     addCaTrust(info, req.params.nodeName);
 
@@ -933,6 +935,8 @@
     let info = url.parse(viewUrl);
     info.path = encodeURI(`${Config.basePath(node)}${path}`);
     info.agent = (client === http ? internals.httpAgent : internals.httpsAgent);
+    // use internals.esQueryTimeout, but convert it to milliseconds
+    info.timeout = parseInt(Config.get("elasticsearchTimeout", 300), 10) * 1000;
     addAuth(info, user, node);
     addCaTrust(info, node);
     let preq = client.request(info, function (pres) {
